app/api/activitylog/specs/activitylog.spec.js

```diff
@@ -47,53 +47,61 @@ describe('activitylog', () => {
       });
     });
 
-    it('should filter by method', async () => {
-      const { rows: entries } = await activitylog.get({ method: ['POST'] });
-      expect(entries.length).toBe(1);
-      expect(entries[0].method).toBe('POST');
-    });
+    describe('filtering', () => {
+      it('should filter by method', async () => {
+        const { rows: entries } = await activitylog.get({ method: ['POST'] });
+        expect(entries.length).toBe(1);
+        expect(entries[0].method).toBe('POST');
```

Comment:
Maybe simplify this code to a foreach so that we don't need to know how many there are, all we need to know is that all of them are POST. This way if the fixtures change the condition is better covered and still met.